src/analysis/retail/mage/frost/core/WintersChill.tsx

```diff
@@ -236,6 +259,91 @@ class WintersChill extends Analyzer {
       </Statistic>
     );
   }
+
+  get guideSubsection(): JSX.Element {
+    const cooldown = {
+      id: -1,
+      name: 'Cooldown',
+      icon: 'inv_misc_questionmark',
+    };
+
+    const wintersChill = <SpellLink spell={SPELLS.WINTERS_CHILL} />;
+    const flurry = <SpellLink spell={TALENTS.FLURRY_TALENT} />;
+    const frostbolt = <SpellLink spell={SPELLS.FROSTBOLT} />;
+    const glacialSpike = <SpellLink spell={TALENTS.GLACIAL_SPIKE_TALENT} />;
+    const iceLance = <SpellLink spell={TALENTS.ICE_LANCE_TALENT} />;
+    const rayOfFrost = <SpellLink spell={TALENTS.RAY_OF_FROST_TALENT} />;
+
+    const cooldownIcon = <SpellIcon spell={cooldown} />;
+    const glacialSpikeIcon = <SpellIcon spell={TALENTS.GLACIAL_SPIKE_TALENT} />;
+    const iceLanceIcon = <SpellIcon spell={TALENTS.ICE_LANCE_TALENT} />;
+    const rayOfFrostIcon = <SpellIcon spell={TALENTS.RAY_OF_FROST_TALENT} />;
+
+    const explanation = (
+      <>
+        <b>{wintersChill}</b> is an extremely important part of playing Frost effectively.
+        <br />
+        There are 2 main rules to follow:
+        <ul>
+          <li>
+            <b>Precast</b>
+          </li>
+          You should cast {this.hasGlacialSpike ? <>{glacialSpike} or </> : ''}
+          {frostbolt} before {flurry}.<br />
+          {this.hasGlacialSpike && (
+            <>
+              At 4 <SpellLink spell={SPELLS.MASTERY_ICICLES} /> you can cast {flurry} without
+              precast.
+              <br />
+              <small>
+                Precast priority order: {glacialSpike} {frostbolt}{' '}
+              </small>
+            </>
+          )}
+          <li>
+            <b>{wintersChill} stacks</b>
+          </li>
+          Consume both {wintersChill} stacks with {this.hasGlacialSpike && <>{glacialSpike} or </>}
+          {iceLance}.<br />
+          {this.hasRayOfFrost && (
+            <>
+              If {rayOfFrost} is available, use it at 1 stack of {wintersChill}.<br />
+            </>
+          )}
+          <small>
+            Priority order: {this.hasRayOfFrost && <>{rayOfFrost} </>}
+            {this.hasGlacialSpike && <>{glacialSpike} </>}
+            {iceLance}
+          </small>
+        </ul>
+        Your rotations should look like this: <br />
+        <SpellSeq spells={[SPELLS.FROSTBOLT, TALENTS.FLURRY_TALENT, cooldown, cooldown]} /> <br />
+        <SpellSeq
+          spells={[TALENTS.GLACIAL_SPIKE_TALENT, TALENTS.FLURRY_TALENT, cooldown, cooldown]}
+        />{' '}
+        <br />
+        filling the {cooldownIcon}s with {this.hasGlacialSpike && <>{glacialSpikeIcon} </>},{' '}
```

Comment:
the leading comma here is outside the JSX for glacial spike, but should be inside it for cases where glacial spike is not taken.

```suggestion
        filling the {cooldownIcon}s with {this.hasGlacialSpike && <>{glacialSpikeIcon},{' '}</>}
```